join.html

```diff
@@ -61,41 +61,45 @@ <h4>Fee Structure</h4>
     <table class="table">
       <tbody>
         <tr>
-          <td><strong>Backer Hacker</strong></td>
+          <td><strong>Standard Member</strong></td>
           <td>
             <ul>
-              <li>$100 per month, no further discounts.</li>
+              <li>$100 per month. Rates can be reduced to $60 per month by becoming a 'Volunteer Hacker.'</li>
             </ul>
           </td>
         </tr>
         <tr>
-          <td><strong>Standard Hacker</strong></td>
+          <td><strong>Volunteer Member</strong></td>
           <td>
             <ul>
-              <li>$60 per month, or</li>
-              <li>$55 per month when paying 3 months or more in advance (up to 12 months)</li>
+              <li>$60 per month. </li>
+			  <li>To be on this rate you must become a volunteer by signing an agreement. Each month you fulfill your
+			  agreement we will discount your membership fees to this level. Don't be shy! There's always plenty to do.</li>
             </ul>
           </td>
         </tr>
         <tr>
-          <td><strong>Starving Hacker</strong></td>
+          <td><strong>Concessional Volunteer Member</strong></td>
           <td>
             <ul>
               <li>$30 per month, no further discounts</li>
-              <p>To be eligible for this level you must be genuinely unable to afford the standard tier. Being a
-                student or unemployed generally makes you eligible. You will require proof of eligibility during your
-                induction.</p>
+			  <li>To be on this rate you must become a volunteer by signing an agreement. Each month you fulfill your
+			  agreement we will discount your membership fees to this level.</li>
+			  <li>The volunteer workload for this rate is higher than for the 'Volunteer Member' rate. You can fulfill these expectations by 
+			  attending the monthly working bee.</li>
+              <li>To be eligible for this rate you must be genuinely unable to afford the standard tier. Being a
+                student or unemployed generally makes you eligible. Proof of eligibility will be required during the sign-up process.</li>
             </ul>
           </td>
         </tr>
         <tr>
-          <td><strong>Lifetime Hacker</strong></td>
+          <td><strong>Lifetime Member</strong></td>
           <td>
             <ul>
               <li>$0 per month, for life</li>
             </ul>
             <p>
-            Available to any member having paid 10 years membership. As a thankyou to those who have
+            Available to any member having paid 10 years membership. As a thank you to those who have
```

Comment:
Do we need to specify that its now either 10years of legacy membership or 10years of volunteering?